### DISCUSSION TITLE
Creating onnx file from openai/whisper-large-v3

### DISCUSSION THREAD

**Discussion by arthurvb** [microsoft/Olive#948] [General]
Dear All,

To save you time just some notes how I got Olive to work for `openai/whisper-large-v3`. 
- I had to add `"save_as_external_data": true` in the json (e.g. `whisper_cpu_int8.json`) in  conversion/config and in prepost/config
- In `olive-env/lib/python3.10/site-packages/olive/passes/onnx/insert_beam_search.py` I removed the check of the onnx model by changing the last line to `return model_proto_to_olive_model(combined_model, output_model_path, config, False)` . Otherwise the process stopped with `The model does not have an ir_version set properly.`

Hopefully this will help some of you,

Arthur

**Comment by thewh1teagle** [comment]
Hey @arthurvb
First of all thanks a lot for keeping the notes here.

Did you succeed to run tiny / medium models as well?
I tried to follow the repo but it output only empty result.
Maybe you can see if I'm missing something?
https://github.com/microsoft/Olive/issues/1291

**Comment by xiaoyu-work** [comment]
closing as whisper model is supported here: https://github.com/microsoft/olive-recipes
